docs/history.rst

```diff
@@ -0,0 +1,12 @@
+History
```

Comment:
I thought the titles should not be in the separate files.

Reply:
There should not be section titles in the rst fragments that are included in the README. My reasoning is that we want to avoid the README to become too complex, and avoid section level conflicts when including the fragments in a sphinx doc. Here we are in the sphinx doc where the author controls everything so there are no such constraints.

Note HISTORY.rst is not included in the README and for that one I kept section titles in the [example](https://raw.githubusercontent.com/acsone/mis-builder/10.0-generated-readme/mis_builder_budget/HISTORY.rst).